mesa/experimental/components/matplotlib.py

```diff
@@ -55,6 +55,8 @@ def portray(g):
             out["c"] = c
         return out
 
+    space_ax.set_xlim(-1, space.width)
```

Comment:
This has to start at -1, because otherwise the markers at position 0 are cropped out.

Reply:
Probably go with `space.width+1` as well for the same reason?

and for `_draw_continuous_space` (which is were my problem started)

```python
space_ax.scatter(**portray(space))
space_ax.set_xlim(xmin=space.x_min, xmax=space.x_max)
space_ax.set_ylim(ymin=space.y_min, ymax=space.y_max)
```

Reply:
`space.width+1` is not necessary, because the position is 0-index based, so technically it already has +1 in it by doing `space.width` instead of `space.width-1`.